.gitignore

```diff
@@ -3,6 +3,10 @@
 .ccls-cache
 compile_commands.json
 tags
+Makefile
+CMakeCache.txt
+CMakeFiles/
+cmake_install.cmake
```

Comment:
Please keep this block alphabetically sorted.

Reply:
i'll use ascii sorting